CRM/Upgrade/Incremental/Base.php

```diff
@@ -182,6 +182,51 @@ public static function addColumn($ctx, $table, $column, $properties, $localizabl
   }
 
   /**
+   * Modify a column in a table
+   *
+   * @param CRM_Queue_TaskContext $ctx
+   * @param string $table
+   * @param string $column
+   * @param string $properties
+   * @param bool $localizable is this a field that should be localized
+   * @param string $newColumn (new name for column)
+   * @return bool
+   */
+  public static function alterColumn($ctx, $table, $column, $properties, $localizable = FALSE, $newColumn = NULL) {
+    if (!isset($newColumn)) {
+      // Option to rename column
+      $newColumn = $column;
+    }
+    $domain = new CRM_Core_DAO_Domain();
+    $domain->find(TRUE);
+    $queries = array();
+    if (!CRM_Core_BAO_SchemaHandler::checkIfFieldExists($table, $column)) {
```

Comment:
Note that we may need new param on that function to say if we should be checking for translated name or not